py/src/ai/h2o/sparkling/ml/algos/H2OAutoML.py

```diff
@@ -68,8 +71,16 @@ def __init__(self,
         self._set(**kwargs)
 
     def leaderboard(self):
+        warnings.warn("The 'leaderboard' method is deprecated and will be removed in the SW version 3.30. "
+                      "Use the 'getLeaderboard' method instead!")
         leaderboard_java = self._java_obj.leaderboard()
         if leaderboard_java.isDefined():
-            return DataFrame(leaderboard_java.get(), self._hc._sql_context)
+            return DataFrame(leaderboard_java.get(), SparkSession._instantiatedSession._wrapped)
```

Comment:
Fixing the current `leaderboard` implementation

Reply:
👍

Reply:
I'm going to merge this change, just will change ``SparkSession._instantiatedSession`` to ``SparkSession.builder.getOrCreate()``. It is possible that user is using just spark context and the session does not exist.

It should work in a same way, if you have any comments, please let me know :)